control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -169,6 +169,8 @@ message PublishVolumeRequest {
   optional string target_node = 2;
   // share protocol
   optional VolumeShareProtocol share = 3;
+  // republish the volume on the node, without removing existing nexus if any
```

Comment:
```suggestion
  // allows republishing the volume on the node by shutting down the existing target first
  // once the new target is ready you must destroy the previous target
```

Reply:
Actually I'm starting to wonder if we should have a republish message, which fails if the volume is not currently published and maybe returns the old target uuid so that we may provide the cluster agent a uuid to destroy old targets, wdyt?

Reply:
That's a very good suggestion. This will take some undoing of the things in this PR.

Reply:
> by shutting down the existing target first

Should we add this? This step will be taken care of by the cluster agent, so would not be a part of publish call

Reply:
Isn't shutdown done by the core agent during republish?

Reply:
I think the cluster agent does that. It shut downs the nexus and then make a republish call.

Reply:
I'm not sure that sounds right, the cluster agent does not own the nexus, it doesn't even know its uuid, only it's name.

Reply:
Can it not get it from the volume spec? I was told the cluster agent drives all the steps.